composer.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "matomo-org/matomo-coding-standards",
+    "description": "PHP CodeSniffer ruleset implementing the Matomo coding standards extending PSR12 coding standards.",
```

This repo will grow to also include front-end CS and other rules, so we should describe the package in a generic way.
```suggestion
    "description": "Coding standards for Matomo",
```

---

@michalkleiner Makes sense :+1: , updated as suggested